File: Android Kotlin Retrofit Node Express- Orphanslife/app/src/main/java/com/kaligotla/orphanslife/model/Location.kt
package com.kaligotla.orphanslife.model

import com.google.gson.annotations.SerializedName

data class Location(
    @SerializedName("location_id")
    val location_id: Int,

    @SerializedName("pincode")
    val pincode: Int,

    @SerializedName("area")
    val area: String,

    @SerializedName("city")
    val city: String,

    @SerializedName("district")
    val district: String,

    @SerializedName("state")
    val state: String
)

File: MongoDB Express - API - Orphanslife/Data/model/Location.js
const mongoose = require('mongoose')
const Schema = mongoose.Schema
const passportLocalMongoose = require('passport-local-mongoose');

var Location = new Schema({
    location_id:{
        type: Number
    },
    pincode:{
        type: Number
    },
    area:{
        type: String
    },
    city:{
        type: String
    },
    district:{
        type: String
    },
    status:{
        type: String
    }
}, { collection: 'location' })

Location.plugin(passportLocalMongoose);
  
module.exports = mongoose.model('Location', Location)